.gitignore

```diff
@@ -41,6 +41,7 @@ tmp/
 *.bak
 *.swp
 *~.nib
+desktop/src/main/resources
```

Comment:
Unsure why this is (still) in here :-)

Reply:
That’s there because there something wrong with the path for resources, and I didn’t want it to get pushed in the PR.

Reply:
Oh you can just choose not to commit those files if your workspace has gone weird. The path itself is perfectly valid - there just aren't any actual resources for the desktop sub-project (yet). So we shouldn't gitignore it.